cli/tests/default/e2e/test_test.py

```diff
@@ -56,6 +94,7 @@ def test_cli_test_yaml_language(run_semgrep_in_tmp: RunSemgrep, snapshot):
     )
 
 
+# ??
```

Comment:
This test checks that files with multiple suffixes (ie multiple dots) in the file names are included. (https://github.com/semgrep/semgrep/commit/6c07e59f16334e943bb2b0165ad3386e7909b12f)

Reply:
ok. Thx. Will fix the comment.